1-js/10-error-handling/1-try-catch/article.md

```diff
@@ -1,418 +1,417 @@
-# Error handling, "try..catch"
+# Obsługa błędów, "try...catch"
 
-No matter how great we are at programming, sometimes our scripts have errors. They may occur because of our mistakes, an unexpected user input, an erroneous server response, and for a thousand other reasons.
+Nieważne jak świetnymi jesteśmy programistami, zdarza się, że do naszych programów wkradają się błędy. Mogą wystąpić jako rezultat literówki, przez niespodziewane dane wejściowe użytkownika, błędnej odpowiedzi serwera czy z tysiąca wielu innych powodów. 
 
-Usually, a script "dies" (immediately stops) in case of an error, printing it to console.
+Z reguły, nasz program zostaje natychmiastowo przerwany w sytuacji napotkania błędu, zaraz po zwróceniu jego zawartości do konsoli.
 
-But there's a syntax construct `try..catch` that allows to "catch" errors and, instead of dying, do something more reasonable.
+Przy użyciu instrukcji `try...catch`, możemy obsłużyć błędy w bardziej sensowny sposób.
 
-## The "try..catch" syntax
+## Składnia "try...catch"
 
-The `try..catch` construct has two main blocks: `try`, and then `catch`:
+Instrukcja `try...catch` zawiera dwa główne bloki, `try {...}` oraz `catch(err) {...}`:
 
 ```js
 try {
 
-  // code...
+  // instrukcje do wykonania...
 
 } catch (err) {
 
-  // error handling
+  // obsługa błędów z pierwszego bloku...
 
 }
 ```
 
-It works like this:
+Sposób działania:
 
-1. First, the code in `try {...}` is executed.
-2. If there were no errors, then `catch(err)` is ignored: the execution reaches the end of `try` and goes on, skipping `catch`.
-3. If an error occurs, then `try` execution is stopped, and the control flows to the beginning of `catch(err)`. The `err` variable (can use any name for it) will contain an error object with details about what happened.
+1. Rozpoczęcie wykonywania się kodu zawartego w bloku `try {...}`.
+2. Bezbłędne wykonanie pierwszego bloku oznacza, że drugi blok `catch(err) {...}` zostaje pominięty, a program rusza dalej.
```

Comment:
To zdanie brzmi dość skomplikowanie, nie lepiej "jeśli błąd nie wystąpi"?
Strona bierna utrudnia rozumienie, w tutorialach starajmy się jej unikać